content/en/news/releases/1.10.x/announcing-1.10/_index.md

```diff
@@ -0,0 +1,78 @@
+
+For example, imagine you have two revisions, `1-7-6` and `1-8-0`. You create a revision tag `prod` pointed to revision `1-7-6` and create a revision tag `canary` pointed to the newer `1-8-0` revision.
+
+{{< image width="40%"
+    link="/docs/setup/upgrade/canary/revision-tag-1.png"
+    caption="Namespaces A and B pointed to 1-7-6, namespace C pointed to 1-8-0"
+    >}}
+
+Now, when you’re ready to promote the `1-8-0` revision from `canary` to `prod`, you can re-associate the `prod` tag with the `1-8-0` Istiod revision. Now all namespaces using `istio.io/rev=prod` will use the newer `1-8-0` revision for injection.
+
+{{< image width="40%"
+    link="/docs/setup/upgrade/canary/revision-tag-2.png"
+    caption="Namespaces A, B, and C pointed to 1-8-0"
+    >}}
+
+Check out the [updated Canary Upgrade guide](/docs/setup/upgrade/canary/#stable-revision-labels-experimental) for a walk-through you can follow along with!
+
+## Sidecar Networking Changes
+
+In previous Istio releases, Istio has rewritten pod networking to trap traffic from `eth0` and send it to applications on `lo`. Most applications bind to both interfaces and don’t notice any difference; however some applications are specifically written to only expect specific traffic on either interface (e.g. it’s common to expose admin endpoints only on `lo` and never over `eth0`, or for stateful applications to bind only to `eth0`). These applications’ behavior can be impacted by how Istio directs traffic into the pod.
+
+In 1.10, Istio is updating Envoy to send traffic to the application on `eth0` rather than `lo` by default. For new users, this should only be an improvement. For existing users, `istioctl experimental precheck` will identify pods that listen on localhost and may be impacted as IST0143.
+
+See [the write-up](/blog/2021/upcoming-networking-changes/) by John Howard for a more in depth overview of the change, how and why it might impact you, and how to preserve today’s behavior to enable a seamless migration.
+
+## A Fresh Look for Istio.io
+
+We’ve revamped Istio.io with a totally new look! This is the first major change to Istio’s site since the project launched nearly four years ago (we’ll celebrate that anniversary on May 24th!). We hope these changes help make the site more user-friendly, easier to navigate, and more readable overall.
+
+This effort was sponsored by Google Cloud and we want to send a special thanks to [Craig Box](https://twitter.com/craigbox), [Aizhamal Nurmamat kyzy](https://twitter.com/iamaijamal) and [Srinath Padmanabhan](https://twitter.com/srithreepo) for driving this effort, and to all the folks that helped review and provide feedback to early revisions.
+
+Please give us any feedback you have by filing an issue on the [istio.io repository](https://github.com/istio/istio.io).
+
+## Deprecations
```

Comment:
```suggestion
## Opening up our design docs

Istio's working groups keep their design documents and planning artifacts in a community Google Drive. Anyone is welcome to view these documents, but due to Istio's heritage inside Google, you first had to [join a public group](https://groups.google.com/g/istio-team-drive-access) to get access to the drive and the files within.

On the 20th of May we will change the Drive so that the files are accessible to everyone on the Internet, without having to log in, or join the group. This change will make sharing Istio technical documentation easier. and is another way we're ensuring Istio is a fully open project.

Files will remain at the same URLs as before, but the Drive and its subfolders will change URLs. If you are a regular user of the Community Drive, please be aware of this change.  All contributors and Drive members will be contacted this week with the new details.

## Deprecations
```